primitives/version/src/lib.rs

```diff
@@ -103,7 +103,7 @@ pub struct RuntimeVersion {
 	/// index.
 	///
 	/// It need *not* change when a new module is added or when a dispatchable is added.
-	pub transaction_version: u32,
+	pub transaction_version: u8,
```

Comment:
Nope, this will break all chains. Please revert.

Reply:
Will do.  Is the rest of the code okay?